--- shartel\charP.d	Thu Oct 16 23:48:54 2003
+++ C:\BWP Patchstudio\patched files\shartel\charP.d	Sun Sep 09 09:05:59 2012
@@ -2,32 +2,32 @@
 
 
 IF ~Global("sharteelJoinedParty","GLOBAL",1) Global("sharteelKickedOut","GLOBAL",0)~ THEN BEGIN Raus0
-	SAY ~Was ist, <CHARNAME>?~ 
-	IF ~~ THEN REPLY ~Es ist nichts, bleibt in der Gruppe.~ DO ~JoinParty()~ EXIT
-	IF ~~ THEN REPLY ~Wird Zeit, dass wir getrennte Wege gehen.~ GOTO Raus1
+	SAY @0 
+	IF ~~ THEN REPLY @1 DO ~JoinParty()~ EXIT
+	IF ~~ THEN REPLY @2 GOTO Raus1
 END
 
 
 IF ~Global("sharteelJoinedParty","GLOBAL",0) Global("sharteelKickedOut","GLOBAL",1)~ THEN BEGIN Auf0
-	SAY ~Ich hätte nicht gedacht, dass ich Euch noch einmal so schnell wiedersehen würde, <CHARNAME>. Kann ich Euch irgendwie helfen?~
-	IF ~~ THEN REPLY ~Wenn Ihr Interesse habt, dann schließt Euch mir wieder an, Shar-Teel.~ GOTO Auf1
-	IF ~~ THEN REPLY ~Im Augenblick habe ich keine Verwendung für Euch.~ GOTO Raus2
+	SAY @3
+	IF ~~ THEN REPLY @4 GOTO Auf1
+	IF ~~ THEN REPLY @5 GOTO Raus2
 END
 
 
 IF ~~ THEN BEGIN Auf1
-	SAY ~So ist es. Ziehen wir wieder gemeinsam in die Schlacht! Hinein in unendliches Blutvergießen - möge das Schreien unserer Feinde das Lied unsereres Triumphes sein!~
+	SAY @6
 	IF ~~ THEN DO ~SetGlobal("sharteelJoinedParty","GLOBAL",1) SetGlobal("sharteelKickedOut","GLOBAL",0) JoinParty()~ EXIT
 END
 
 
 IF ~~ THEN BEGIN Raus1
-	SAY ~Wenn Ihr das wollt, <CHARNAME>... es war irgendwie eine angenehme Zeit mit Euch...zumindest gab es immer genug Blutvergießen in Eurer Nähe. Wie dem auch sei - lebt wohl. Vielleicht sehen wir uns ja sogar eines Tages wieder.~
+	SAY @7
 	IF ~~ THEN DO ~SetGlobal("sharteelJoinedParty","GLOBAL",0) SetGlobal("sharteelKickedOut","GLOBAL",1) EscapeAreaMove("AR0406",681,1094,0)~ EXIT
 END
 
 
 IF ~~ THEN BEGIN Raus2
-	SAY ~Ich werde noch eine Weile hier in der Kupferkrone bleiben. Ist gar nicht so schlecht hier. Wenn man Streit sucht, wird man hier jedenfalls fündig und es gibt hier genug dämlicher Kerle, denen ich die Zähne einschlagen kann, wenn sie sich auf einen Arena-Kampf einlassen...~
+	SAY @8
 	IF ~~ THEN EXIT
 END
