--- PoFQuestPack\English\D\FKIN2MOD.tra	Mon Apr 26 14:32:19 2021
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\FKIN2MOD.tra	Mon Mar 13 08:37:13 2023
@@ -15,7 +15,7 @@
 @15  = ~It's him, we have to go there now. Or I can go alone, you can't come with me.~
 @16  = ~You helped me, so I'll help you.~
 @17  = ~Thank you very much, King, we're leaving immediately.~
-@18  = ~Potøebuju si ještì nìco zaøídid pøed tou cestou, zatím pøipravte vše potøebné.~
+@18  = ~I have something else to do before the journey. You can prepare whatever you need for now.~
 @19  = ~Everything is prepared... You can.~
 @20  = ~On the way to death.
 
