--- PoFQuestPack\English\Baf_new\PF0148.tra	Mon Apr 26 14:00:25 2021
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\Baf_new\PF0148.tra	Mon Mar 13 08:46:13 2023
@@ -3,8 +3,8 @@
 @2    = ~With the help of Targu, we stopped the drow invasion. But they united with the barbarians, who had long sought to destroy the cities of the North. Thousands of them came here, we resist as much as we can, but we do not have so much strength. Our only hope is you. Lady Aribet will tell you what your task will be.~
 @3    = ~Yes, as you have heard, the situation is serious, our only chance is to get to Helm Fortress unnoticed and kill the priest who runs them. Then maybe they'll back off. They are everywhere in the city, as you go out, be careful, they are very dangerous. At the east gate you will find our friend who confesses the best in the district and he will take you from town to Fort Helm. I wish you good luck to adventurers.~
 @4    = ~Do you understand? Do you need anything else?~
-@5    = ~Ne to ne je mi to jasné, jen ne budu pryè, zkuste kontaktovat mé pøátelé a se ke mnì pøipojí v Helmovì tvrzi. Ještì bych potøeboval nìjaké vybavení.~
-@6    = ~Tady mùj pøítel nalevo ode mnì t dá co budeš potøebovat. I'll try to find your friends. Thank you for your help.~
+@5    = ~No, no, I don't understand, but before I go, try to contact my friends and they will join me in Helm's Keep. I would still need some equipment.~
+@6    = ~Here my friend to my left will give you what you need. I'll try to find your friends. Thank you for your help.~
 @7    = ~When you need a gun, you can come to me.~
 @8    = ~All right, I like it, I'll try to do whatever I can for your town. Goodbye.~
 @9    = ~One more thing. I called three of my best people. I think I definitely need help until you get back with your friends.~
